jupyter/restore_raft_quorum_with_tablets.ipynb

```diff
@@ -0,0 +1,680 @@
+   "id": "102234be",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "voters_status = get_current_voters_states(cluster.nodes, cluster.nodes[0])\n",
+    "alive_dc_region, dead_dc_region = sorted(voters_status, key=lambda region: len(list(filter(lambda v: v.is_voter, voters_status[region]))))\n",
+    "nodes_by_region = cluster.nodes_by_region(cluster.nodes)\n",
+    "host_id_node_map = {node.host_id: node for node in cluster.nodes}\n",
+    "dead_node_host_ids_maps = {node.host_id: node for node in nodes_by_region[dead_dc_region]}\n",
+    "dead_node_host_ids = list(dead_node_host_ids_maps.keys())\n",
+    "verification_node = nodes_by_region[alive_dc_region][0]\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d99313b3",
+   "metadata": {},
+   "source": [
+    "### Stop all nodes in DC with largest number of voters.\n",
+    "\n",
+    "Need stop them very fast better in parallel, so voters was not reassigned to another dc\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e37a5997",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "parallel_stop = ParallelObject(nodes_by_region[dead_dc_region], num_workers=len(nodes_by_region[dead_dc_region]), timeout = 600)\n",
+    "parallel_stop.run(stop_scylla_server_not_gently)\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "127443ea",
+   "metadata": {},
+   "source": [
+    "# GET CLUSTER STATUS\n",
+    "\n",
+    "This cell could be used to validate current status of cluster with `nodetool status`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "58689968",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "verification_node = nodes_by_region[alive_dc_region][0]\n",
+    "print(verification_node.run_nodetool(\"status\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de494749",
+   "metadata": {},
+   "source": [
+    "### Validate that raft quorum lost\n",
+    "Try to remove node dead node\n",
+    "Call read barrier.\n",
+    "\n",
+    "Both operations should failed with exception"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c479fbfb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "try:\n",
+    "    verification_node.run_nodetool(f\"removenode {dead_node_host_ids[0]} --ignore-dead-nodes {','.join(dead_node_host_ids[1:])}\")\n",
+    "except Exception as exc:\n",
+    "    print(f\"Expected to fail. Error: {exc}\")\n",
+    "\n",
+    "try:\n",
+    "    verification_node.raft.call_read_barrier()\n",
+    "except Exception as exc:\n",
+    "    print(f\"Expected to fail. Error: {exc}\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "571d6f73",
+   "metadata": {},
+   "source": [
+    "# Recovery procedure\n",
+    "Execute each step one by one\n",
+    "Use 'GET CLUSTER STATUS' cell to check cluster status between steps if need"
```

Comment:
`\n` doesn't cause the new line to appear in the notebook view, I see:
```
Execute each step one by one Use 'GET CLUSTER STATUS' cell to check cluster status between steps if need
```

There are more places like this.